### PR TITLE
Feature: Wipe database info screen

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Create an info screen for wiping database

### TODO:
- [x] connect to unlock screen
- [x] show confirm dialog
- [ ] connect to wipe complete screen
- [ ] wipe action

### CODE REVIEW

**Comment by johnxnguyen** [comment]
Also here there may be copy adjustments.

**Comment by billypchan** [comment]
@johnxnguyen We can merge this PR first and start design review for the UI styling first.

**Comment by johnxnguyen** [comment]
Are the todo items in the PR description still to be completed?

**Comment by billypchan** [comment]
The Todos will be included in the coming PRs.

File: Wire-iOS/Sources/Helpers/ColorScheme.swift [hunk 1]
```diff
@@ -249,13 +249,16 @@ final class ColorScheme: NSObject {
...
-    }
-    
-    func color(named: ColorSchemeColor, variant: ColorSchemeVariant) -> UIColor {
+    func color(named: ColorSchemeColor, variant: ColorSchemeVariant? = nil) -> UIColor {
+        let colorSchemeVariant: ColorSchemeVariant
+        if let variant = variant {
+            colorSchemeVariant = variant
+        } else {
+            colorSchemeVariant = self.variant
+        }
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/Helpers/ColorScheme.swift:258]:
```suggestion
let colorSchemeVariant = variant ?? self.variant
```

File: Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Unlock/UnlockViewController.swift [hunk 2]
```diff
@@ -202,7 +201,9 @@ final class UnlockViewController: UIViewController {
 
     @objc
     private func onWipeButtonPressed(sender: AnyObject?) {
-        //TODO push wipe screen
+        // push wipe screen
+
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Unlock/UnlockViewController.swift:205]: This comment is redundant and should be removed.

File: Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Unlock/UnlockViewController.swift [hunk 3]
```diff
@@ -202,7 +201,9 @@ final class UnlockViewController: UIViewController {
 
     @objc
     private func onWipeButtonPressed(sender: AnyObject?) {
-        //TODO push wipe screen
+        // push wipe screen
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Unlock/UnlockViewController.swift:204]:
```suggestion
```

**Pull request merged**